### DISCUSSION TITLE
ideal number of bootnodes for a 70 validator PoS network?

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#1188] [General]
# [ Subject of the issue ]
## Description
Good afternoon, is there an optimum number of bootnodes for a 70 validator network. Would 10 bootnodes be enough?

## Your environment
- OS and version. ubuntu 20.04
- The version of the Polygon Edge.    0.6.1
  (*Confirm the version of your Polygon edge client by running the following command: `polygon-edge version --grpc-address GRPC_ADDRESS`*)
- The branch that causes this issue.
- Locally or Cloud hosted (which provider). cloud
- Please confirm if the validators are running under containerized environment (K8s, Docker, etc.). no

## Steps to reproduce
- Tell us how to reproduce this issue.
- Where the issue is, if you know.
- Which commands triggered the issue, if any.
- Provide us with the content of your genesis file.
- Provide us with commands that you used to start your validators.
- Provide us with the peer list of each of your validators by running the following command: `polygon-edge peers list --grpc-address GRPC_ADDRESS`.
- Is the chain producing blocks and serving customers atm?

## Expected behavior
- Tell us what should happen.
- Tell us what happened instead.

## Logs
Provide us with debug logs from all of your validators by setting logging to `debug` output with: `server --log-level debug`

## Proposed solution
If you have an idea on how to fix this issue, please write it down here, so we can begin discussing it
 @laviniat1996 could you help please?

**Comment by laviniat1996** [comment]
Hi @Nexsuperne101 ,

I have converted this to a discussion.

**Comment by laviniat1996** [comment]
@Nexsuperne101 To me, 10 bootnodes seems like a decent number. I would probably add a few more. Let me check with the team and will get back with a more concise reply.

**Reply by laviniat1996** [reply]
@Nexsuperne101 I'm back. Ten bootnodes should be enough indeed.

**Reply by ghost** [reply]
@laviniat1996 thank you very much for your reply.  I will let the boss know 👍
Thanks, Nex
